app/assets/stylesheets/main.scss

```diff
@@ -82,6 +82,24 @@
   .grader_distribution {
     margin-bottom: 0;
   }
+
+  #grader-right{
+    height:380px;
+    overflow:scroll;
+
+  }
+
+  ::-webkit-scrollbar {
+    -webkit-appearance: none;
+    width: 7px;
+  }
+
+  ::-webkit-scrollbar-thumb {
+    border-radius: 4px;
```

Comment:
Properties should be ordered background-color, border-radius, -webkit-box-shadow